package-lock.json

```diff
@@ -4,6 +4,314 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@google-cloud/common": {
```

Comment:
Was this file accidentally committed?

Reply:
It showed up as merge conflict after your code changes. So I copied your current version of package-lock and pasted it in my code.